packages/xod-client/src/core/styles/components/PatchSVG.scss

```diff
@@ -1,4 +1,13 @@
 .PatchSVG {
   cursor: default;
   user-select: none;
+
+  &.grab {
+    cursor: move;
+    cursor: grab;
+  }
+
+  &.grab:active {
+    cursor: grabbing;
+  }
```

Comment:
Wouldn’t it be more consistent to use single term pan/panning? Now we have grab and pan that are used as synonyms.